src/certificates.ts

```diff
@@ -15,7 +15,8 @@ const debug = createDebug('devcert:certificates');
  * individual domain certificates are signed by the devcert root CA (which was
  * added to the OS/browser trust stores), they are trusted.
  */
-export default async function generateDomainCertificate(domain: string): Promise<void> {
+export default async function generateDomainCertificate(domains: string | string[]): Promise<void> {
+  const domain = Array.isArray(domains) ? domains[0] : domains;
```

Comment:
This is an example of the kind of code you can simplify if you cast a request for a single domain into an array of length 1 earlier in the process.

Reply:
Good idea!